openmc/geometry.py

```diff
@@ -58,6 +58,13 @@ def __init__(
                     univ.add_cell(cell)
                 self._root_universe = univ
 
+    def __contains__(self, cell):
+        if self._root_universe is None:
+            return True
+        else: 
+            return cell in self._root_universe
```

Comment:
Also why are you accessing `_private` object directly instead of through its `@property`?